src/components/Tabs/Annotation/AnnotationDisplay/AnnotationDisplayList.tsx

```diff
@@ -4,7 +4,8 @@ import { Color } from "three";
 
 import { Dataset, Track } from "../../../../colorizer";
 import { getEmptyLookupInfo, getTrackLookups, LookupInfo } from "../../../../colorizer/utils/annotation_utils";
-import { FlexColumn, FlexRowAlignCenter } from "../../../../styles/utils";
+import { FlexColumn, FlexRow, FlexRowAlignCenter } from "../../../../styles/utils";
```

Comment:
The majority of the changes happened in this component. I removed the old card layout and added titles to the two column sections:
![image](https://github.com/user-attachments/assets/f77873bf-7e22-4ef3-95d4-8b923f836a40)